data/src/test/java/com/example/android/data/repositories/CreditRepositoryTest.kt

```diff
@@ -0,0 +1,70 @@
+package com.example.android.data.repositories
+
+import com.example.android.data.sources.RemoteCreditDataSource
+import com.example.android.domain.result.DataResult
+import com.example.android.mocks.mockedCredit
+import com.example.android.mocks.mockedMovie
+import com.nhaarman.mockitokotlin2.whenever
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.junit.MockitoJUnitRunner
+import java.io.IOException
+
+@RunWith(MockitoJUnitRunner::class)
+class CreditRepositoryTest {
+
+    @Mock
+    lateinit var remoteCreditDataSource: RemoteCreditDataSource
+
+    lateinit var creditRepository: CreditRepository
+
+    @Before
+    fun setUp() {
+        creditRepository = CreditRepository(remoteCreditDataSource)
+    }
+
+
+    @Test
+    fun `is getCreditList by remote data source success`() {
```

Comment:
```suggestion
    fun `getCreditList should return expected Success list of credit with given movie id`() {
```

Reply:
* I improved test names.
* I replace whenever with given.
* I used Hamcrest for assertThat.
* For now, I prefer keep these blank spaces because it's easy to me read it.

Reply:
Got it!